### DISCUSSION TITLE
Linux for graphic content

### DISCUSSION THREAD

**Discussion by divtodiv** [linux-surface/linux-surface#820] [General]
Hello, i have a surface book 2 with windows 11 installed, I was planning to move on fedora 36 (100% faster than windows on this device) using the Linux surface kernel. The only issue I know I will face is that I would like to use the adobe suite.

What I should do?

- Install a VM and run windows only for Adobe Suite?

- Or simply give up cause it won't perform at all..

I'm open for any solutions

**Comment by qzed** [comment]
A VM will likely not give you (good) pen and touch support. If you really do need the Adobe Suite, you'll probably have to dual boot and/or stick with Windows.

**Comment by ghost** [comment]
Check out docker and wine?
https://medium.com/@jgautheron/photoshop-in-a-docker-container-285ca0f0966c
Docker on linux is not a vm like on windows/mac
